File: Java/retangulo/src/application/programa.java
package application;

import java.util.Scanner;

import entities.Retangulo;

public class programa {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite o valor da altura do retângulo: ");
		double altura = sc.nextDouble();

		System.out.println("Digite o valor da largura do retângulo: ");
		double largura = sc.nextDouble();
		
		Retangulo retangulo = new Retangulo(altura, largura);
		
		System.out.println(retangulo);
	}

}

File: PHP/retangulo/src/programa.php
<?php

require __DIR__ . '/../vendor/autoload.php';

use Deivz\Retangulo\entities\Retangulo;

echo("Digite o valor da altura do retângulo: " . PHP_EOL);
$altura = readline();

echo("Digite o valor da largura do retângulo: " . PHP_EOL);
$largura = readline();

$retangulo = new Retangulo($altura, $largura);

echo($retangulo->__toString());